src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting/CodeDirectiveOnTypeFormattingTest.cs

```diff
@@ -440,7 +440,8 @@ public async Task Switch_Statment_NestedHtml_NestedCodeBlock()
                 <strong></strong>
             }
             else if (false)
-            {$$
+            {
+1.ToString();$$
```

Comment:
Did these tests previously fail, or are you trying to test something new here?

I don't know whether to be worried that this will regress something :)

Reply:
Ah, so I changed this because the C# on-type formatter doesn't support return characters. But it seems like something is still wrong despite my changes (not with this test, but with the last two tests in the file), so I've been attempting to dig into why.

Reply:
Interesting. So maybe this test was just invalid?

Might be worth doubling checking that it doesn't repro in the real editor in the experimental instance, once you've settled whats going on, just in case.

Reply:
Double checked and it doesn't repro in the real editor, so I think we should be good.

Re: the other failing tests, it appears to be an issue with the C# formatter and is an issue that repros even without the changes from this PR. I filed https://github.com/dotnet/aspnetcore/issues/36390 to track but for now I'll skip those tests.